lib/puppetserver/ca/utils/http_client.rb

```diff
@@ -196,7 +196,11 @@ def self.check_server_online(settings, logger)
             if e.wrapped.is_a? Errno::ECONNREFUSED
               return false
             else
-              raise e
+              if suppress_error
+                logger.inform("Logging a suppressed error: #{e.wrapped}")
```

Comment:
Perhaps
```
Could not determine that puppetserver is offline, connection check failed with #{e.wrapped}. Proceeding with cert signing.
```

Reply:
Aren't we looking for lunch spot, why are you reviewing my PR  🤣